BlueShift-iOS-SDK/BlueshiftEventAnalyticsHelper.m

```diff
@@ -52,10 +51,13 @@ + (NSDictionary *)pushTrackParameterDictionaryForPushDetailsDictionary:(NSDictio
             [pushTrackParametersMutableDictionary setObject:urlElement forKey: kNotificationURLElementKey];
         }
         
-        if([notificationType isEqualToString:kNotificationKey] || [notificationType isEqualToString:kNotificationTypeActionable]) {
-            // If pushDeepLinkURL is nil, then reassign url from pushDetailsDictionary
-            if (![self isNotNilAndNotEmpty:pushDeepLinkURL]) {
-                pushDeepLinkURL = [self getValueBykey: pushDetailsDictionary andKey: kNotificationURLElementKey];
+        if([notificationType isEqualToString:kNotificationKey]) {
+            // If link inside the clk_url is nil, then reassign url from pushDetailsDictionary for key deep_link_url
+            NSString *pushDeepLinkURL = @"";
+            if(urlElement) {
+                pushDeepLinkURL = urlElement;
+            } else {
+                pushDeepLinkURL = [self getValueBykey: pushDetailsDictionary andKey: kPushNotificationDeepLinkURLKey];
```

Comment:
will this cause trouble in the following case?

user clicks on the action button and the button has no URL associated with it. now will this code send clk_url as the deep link of the push notification?

Reply:
thanks for pointing it out. I had fixed it at deep link level, will add fix for this as well.